utils/build_preparation.py

```diff
@@ -23,9 +21,8 @@
 # Re-run this code to bring in any changed files (for incremental build)
 # Be sure to properly specify the source and target directories.
 
-from __future__ import print_function
 import os, sys
-import local_utilities
+from local_utilities import printf, replicate
```

Comment:
Is this change (and related changes below) necessary?  It's more a matter of style.  IMO, the revised code reflects what users may want to use, but in a utility or library module such as this, the previous style makes it easier to diagnose.

Reply:
And, I see I wrote `nxdl2rst.py` in the new way.  So much for my own consistency.